tests/containers/bci_test.pm

```diff
@@ -29,21 +29,27 @@ sub run_tox_cmd {
     my $bci_timeout = get_var('BCI_TIMEOUT', 1200);
     my $bci_reruns = get_var('BCI_RERUNS', 3);
     my $bci_reruns_delay = get_var('BCI_RERUNS_DELAY', 10);
-    my $cmd = "tox -e $env -- -n auto";
+    my $tox_out = "tox_output.txt";
+    my $cmd = "tox -e $env -- -rxX -n auto";
     $cmd .= " -k \"$bci_marker\"" if $bci_marker;
     $cmd .= " --reruns $bci_reruns --reruns-delay $bci_reruns_delay";
+    $cmd .= "| tee $tox_out";
     record_info("tox", "Running command: $cmd");
     my $ret = script_run("timeout $bci_timeout $cmd", timeout => ($bci_timeout + 3));
     if ($ret == 124) {
         # man timeout: If  the command times out, and --preserve-status is not set, then exit with status 124.
         record_soft_failure("The command <tox -e $env> timed out.");
         $error_count += 1;
     } elsif ($ret != 0) {
-        record_soft_failure("The command <tox -e $env> failed.");
+        record_soft_failure("The command <tox -e $env> failed $ret.");
         $error_count += 1;
```

Comment:
A kind question to reviewers: actually all the BCI errors are marked as` Soft Failure` also the effective failures and timeouts: should we remove this status for such cases, instead ?

Reply:
Without an example I cannot make a real judgement here. How would it look?

Reply:
I propose to skip this last question and move it to a possible next discussion, keeping here the only poo requirement as target of this PR.